engines/io_uring.c

```diff
@@ -39,6 +39,7 @@ enum uring_cmd_write_mode {
 	FIO_URING_CMD_WMODE_UNCOR,
 	FIO_URING_CMD_WMODE_ZEROES,
 	FIO_URING_CMD_WMODE_VERIFY,
+	FIO_URING_CMD_WMODE_FLUSH,
```

Comment:
Why is this added?

Reply:
Uhh... my bad.  It's something remaining from the previous approach. Will remove it!

Reply:
Still needs to be removed

Reply:
Will remove it!